net/mwan3/Makefile

```diff
@@ -8,9 +8,9 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mwan3
-PKG_VERSION:=2.0
-PKG_RELEASE:=3
-PKG_MAINTAINER:=Jeroen Louwes <jeroen.louwes@gmail.com>
+PKG_VERSION:=2.0.1
+PKG_RELEASE:=4
```

Comment:
when you increase PKG_VERSION, in general you reset PKG_RELEASE to 1
I haven't checked but if you backported all patches from master, maybe use the same version